helpers/file_helper.rb

```diff
@@ -19,4 +19,13 @@ def self.file_rename(file_name)
     File.rename("files_tmp/#{file_name}", "files_tmp/#{file_new_name}")
     file_new_name
   end
+
+  def self.dir_check_or_create(dir_name)
+    if File.exist? dir_name.to_s
```

Comment:
I think better to use () for arguments, better readability

if File.exist?(dir_name)

And I don't think to_s is really necessary - this method by design receive string as argument